coreloop/watchdogs.c

```diff
@@ -16,4 +20,79 @@ void process_watchdogs (struct core_state* state) {
     for (int i=0; i<4; i++) state->base.TVS_sensors[i] = TVS_sensors_avg[i];
 }
 
+#include <string.h>
+
+void cmd_soft_reset(uint8_t arg);
```

Comment:
`cmd_soft_reset` is implemented in `commanding.c`. Just defined it in the `coreloop.h` and this piece of code should see and use it.